### PR TITLE
Changelog for 2.4.2

### PR DESCRIPTION
**Pull request by daschuer**
Here a step towards overdue 2.4.2 release.

### CODE REVIEW

**Comment by daschuer** [comment]
Updated with latest merges.

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,6 +61,67 @@
 * Fix wrong pitch value on startup, caused by `components.Pot`
   [#11814](https://github.com/mixxxdj/mixxx/issues/11814)
   [#13463](https://github.com/mixxxdj/mixxx/pull/13463)
+* Engine Prime: Fix build-failure [#13397](https://github.com/mixxxdj/mixxx/pull/13397)
+* Engine Prime: Friendlier error message if export fails [#13524](https://github.com/mixxxdj/mixxx/pull/13524)
+* Fix wrong pitch slider value on startup
+  [#11814](https://github.com/mixxxdj/mixxx/issues/11814)
+  [#13463](https://github.com/mixxxdj/mixxx/pull/13463)
```
Comment (ronso0) [inline CHANGELOG.md:68]: Duplicate, it's already listed above
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,14 +4,26 @@
 
 ### Controller Mappings
 
+* Denon MC7000: Fix star up/down logic by only handling button down events [#13588](https://github.com/mixxxdj/mixxx/pull/1358)
```
Comment (ronso0) [inline CHANGELOG.md:7]:
```suggestion
* Denon MC7000: Fix star up/down logic by only handling button down events [#13588](https://github.com/mixxxdj/mixxx/pull/13588)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -49,6 +61,67 @@
...
+  [#13591](https://github.com/mixxxdj/mixxx/pull/13591)
+  [#13590](https://github.com/mixxxdj/mixxx/issues/13590)
+  [#13575](https://github.com/mixxxdj/mixxx/pull/13575)
+* Playlists: Keep correct track selection (# position) when sorting
+  [#13103](https://github.com/mixxxdj/mixxx/pull/13103)
+* Track file export: Various fixes
+  [#13610](https://github.com/mixxxdj/mixxx/pull/13610)
+* Controller engine: Unify/improve logging, expand error dialog's Details box
+  [#13626](https://github.com/mixxxdj/mixxx/pull/13626)
+* Fix quantization in the effect engine (mentronome effect)
```
Comment (ronso0) [inline CHANGELOG.md:100]:
```suggestion
* Fix quantization in the effect engine (metronome effect)
```

**Review by ronso0 — CHANGES_REQUESTED** [review]
Thank you!
Found some nits, didn't yet check the PR/issue links

**Comment by daschuer** [comment]
I have amend the fix. Once this is merged we can proceed with the release. 

Does one want to take care of the remaining bugs before: 
https://github.com/mixxxdj/mixxx/milestones/2.4.2

**Review by Swiftb0y — COMMENTED** [review]
whoops. left this one pending.

**Comment by daschuer** [comment]
@ronso0 Can this be merged now?

**Review by Swiftb0y — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**